### ISSUE TITLE
Standardize filenames of downloadable files

### ISSUE THREAD

**Issue by cytadela8** [warsztatywww/aplikacjawww#464]
Workshop qualification tasks should have a filename with sensible form. Many workshops have filenames like "Zadania-123452.pdf". A better form could be: WWW{edition)-{workshopName}-V{version}.pdf

Same applies for user workshop solutions.

**Comment by MrQubo** [comment]
btw, if I save the file with the name e.g. `README.md` and reupload it again under the same name, then it will be saved under the name `README_somethingrandom.md`.

**Comment by MrQubo** [comment]
Changing the names of solution files might be a bad idea cause the names might be relevant. And the problem with adding a random suffix to filenames also applies. IMO it would be to never change solution file names.

**Comment by cytadela8** [comment]
Yeah, but what if somebody wants to download the solution and have the author in filename? 
Maybe we should solve this by providing a zip download?

**Comment by MrQubo** [comment]
On moodle, you download the zip with all solutions. Each solution is a folder with a name derived from first name, last name, and moodle id. In this folder are all the files uploaded by a student.

**Comment by JanczarKurek** [comment]
I can take that issue :cat:
